shell/components/form/Select.vue

```diff
@@ -139,7 +139,14 @@ export default {
         return get(opt, opt.optionKey);
       }
 
-      return this.getOptionLabel(opt);
+      const label = this.getOptionLabel(opt);
+
+      // label may be type of object
```

Comment:
Do we have known examples of where label may be an object?

Reply:
I was able to get an object out from the test for `MatchExpressions`, where I passed only the options as data.

https://github.com/rancher/dashboard/blob/2bb539c90e3fe3a0a001fadbc038ba3cc4c5560f/shell/components/form/__tests__/MatchExpressions.test.ts